<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SREE" ignored="0" name="Test2" tests="3" failures="0" timestamp="13 Nov 2019 14:22:59 GMT" time="0.127" errors="0">
  <testcase name="@BeforeClass setup" time="0.003" classname="pack1.Test1">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at pack1.Test1.setup(Test1.java:23)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass setup -->
  <testcase name="open_url" time="0.0" classname="pack1.Test1">
    <skipped/>
  </testcase> <!-- open_url -->
  <testcase name="type_data" time="0.0" classname="pack1.Test1">
    <skipped/>
  </testcase> <!-- type_data -->
  <testcase name="tear_down" time="0.0" classname="pack1.Test1">
    <skipped/>
  </testcase> <!-- tear_down -->
</testsuite> <!-- Test2 -->
